### DISCUSSION TITLE
Intro to GitHub Course Thoughts

### DISCUSSION THREAD

**Discussion by HanahLimjoco** [skills/.github#200] [General]
I think that this course is a good introduction for GitHub! I like this course as it is straightforward and easy to follow.

**Comment by Chell0** [comment]
I enjoyed how it was practical and very educative.

**Comment by Learn5Git** [comment]
I am on Module 2, how to migrate repository. Does anyone have the steps on how to do this? I do not see any steps on how to complete this module.

06/02/2023
